seong-geun/app.js

```diff
@@ -115,22 +129,55 @@ app.get("/u_posts", async (req, res) => {
    `,
     [userId]
   );
+
   res.status(200).json({ data: userspost });
-  const { id, profile_image, username, age, email } = req.body;
+});
 
+// 게시글 수정하기
+app.patch("/m_posts", async (req, res) => {
+  const { postingUserId, postingContent } = req.body;
   await appDataSource.query(
-    `INSERT INTO users(
-      id,
-      profile_image,
-      username,
-      age,
-      email
-		) VALUES (?, ?, ?, ?, ?);
-	`,
-    [id, profile_image, username, age, email]
+    `UPDATE
+        posts
+    SET
+        posting_content = ?
+    WHERE
+        posts.user_id = ? 
+    `,
+    [postingContent, postingUserId]
   );
 
-  res.status(201).json({ message: "userCreated" });
+  const m_userspost = await appDataSource.query(
+    `SELECT
+        users.id as userId,
+        users.username as userName,
+        posts.id as postingId,
+        posts.title as postingTitle,
+        posts.posting_content as postingContent
+    
+    FROM 
+        users
+    LEFT JOIN posts
+    ON users.id = posts.user_id
+    `
+  );
+
+  res.status(200).json({ data: m_userspost });
+});
+
+// 게시글 삭제하기
+app.delete("/posts", async (req, res) => {
+  const { postingId } = req.body;
+  await appDataSource.query(
+    `DELETE
+     FROM
+        posts
+     WHERE
+        posts.id = ?
+    `,
+    [postingId]
+  );
+  res.status(200).json({ message: "postingDelete" });
```

Comment:
```suggestion
  res.status(204).send();
```
delete method에도 204라는 status code를 응답으로 더 많이 쓰고 잇습니다. 다만 204는 message로 no content를 의미하고 실제로 바디에 어떠한 데이터를 담아서 보내지 않으므로 send()만 해주면 됩니다.